quartz_solar_forecast/forecasts/v1.py

```diff
@@ -0,0 +1,42 @@
+import pandas as pd
+import xarray as xr
+import os
+from psp.data_sources.nwp import NwpDataSource
+from psp.data_sources.pv import NetcdfPvDataSource
+from psp.serialization import load_model
+from psp.typings import X
+
+dir_path = os.path.dirname(os.path.realpath(__file__))
+
+
+def forecast_v1(nwp_source:str, nwp_xr:xr.Dataset, pv_xr:xr.Dataset, ts:pd.Timestamp, model=None):
```

Comment:
Just thinking going forward, may be useful to have a log of the different forecast versions, what they do/did and how people can use them, and when it was created.

Reply:
Do you mean
1. create a log of models, hence when we upgrade it, its very obvious what has happened
2. add a return column in the predictions that says which model was used and other metadata

Perhaps I could put both of these in new issues

Reply:
I think both of those would be good ideas! (Was thinking more number 1 though)

Reply:
ill add something in the init file for forecasts